src/notifier/notion.py

```
@@ -136,13 +126,96 @@ def _paragraph_block(text: str) -> dict:
+    return [f"**{label}**", content]
+
+
 def _markdown_to_blocks(markdown: str) -> list[dict]:
-    """Markdown 文字列を Notion ブロック辞書のリストに変換して返す。"""
+    """Markdown 文字列を Notion ブロック辞書のリストに変換して返す。
+    Markdown テーブルは Notion table ブロックに変換する。
+    「ラベル：内容」行はラベルと内容を別ブロックに分割する。"""
     blocks = []
-    for line in markdown.splitlines():
+    raw_lines = markdown.splitlines()
+    # ラベル：内容 行を展開してから処理
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)